make-util.js

```diff
@@ -1838,6 +1838,11 @@ var processGeneratedTasks = function(baseConfigToolPath, taskList, makeOptions,
             taskName,
         ];
 
+        if (sprintNumber && parseInt(sprintNumber)) {
```

Comment:
perhaps we need some error message if the argument is invalid, to prevent struggles in debugging the possible problem

Feel free to ignore it

Reply:
added fail message, thx